app-embed-example/setup/metabase-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+echo "seting up $1"
+# get deps
+apk add curl jq
+# get the wait-until script
+curl -L https://raw.githubusercontent.com/nickjj/wait-until/v0.2.0/wait-until -o /usr/local/bin/wait-until && \
+chmod +x /usr/local/bin/wait-until
+# run the script and everything else
+wait-until "echo 'Checking if Metabase is ready' && curl -s http://$1/api/health | grep -ioE 'ok'" 60 && \
+if curl -s http://$1/api/session/properties | jq -r '."setup-token"' | grep -ioE "null"; then echo 'Instance already configured, exiting (or <v43)'; else \
+echo 'Setting up the instance' && \
+token=$(curl -s http://$1/api/session/properties | jq -r '."setup-token"') && \
+echo 'Setup token fetched, now configuring with:' && \
+echo "{'token':'$token','user':{'first_name':'a','last_name':'b','email':'a@b.com','site_name':'metabot1','password':'metabot1','password_confirm':'metabot1'},'database':null,'invite':null,'prefs':{'site_name':'metabot1','site_locale':'en','allow_tracking':'false'}}" > file.json && \
```

Comment:
site_name is twice. Is that correct? Use ex `Embedding Test` as the value.
Don't use real domains in emails, and don't reference `metabot`.

Reply:
I extracted the JSON as it's passed on the setup via the browser, we might have some glitch there